pages/plugin/[[...page]].tsx

```diff
@@ -29,15 +30,42 @@ function PluginLayout(props) {
   )
 }
 
-const { getStaticPaths, getStaticProps } = getStaticGenerationFunctions({
-  strategy: 'fs',
-  localContentDir: CONTENT_DIR,
-  navDataFile: NAV_DATA,
-  product: SOURCE_REPO,
-  githubFileUrl(filepath) {
-    return `https://github.com/hashicorp/${SOURCE_REPO}/blob/${DEFAULT_BRANCH}/${filepath}`
-  },
-})
+const { getStaticPaths: _getStaticPaths, getStaticProps } =
+  getStaticGenerationFunctions({
+    strategy: 'fs',
+    localContentDir: CONTENT_DIR,
+    navDataFile: NAV_DATA,
+    product: SOURCE_REPO,
+    githubFileUrl(filepath) {
+      return `https://github.com/hashicorp/${SOURCE_REPO}/blob/${DEFAULT_BRANCH}/${filepath}`
+    },
+  })
+
+/**
+ * This is a temporary hack to allow multiple `plugin` page components that
+ * may generate duplicate static paths. When `plugin` content is split into
+ * terraform-docs-common, this can be removed.
+ *
+ * - https://nextjs.org/docs/messages/conflicting-ssg-paths
+ */
```

Comment:
I'm actually going to add a _ignoreDirs_ option to `react-docs-page` to avoid this hack.

This would also enable nav-data to not need to be duplicated between `plugin-nav-data` and the 4 new nested nav-data files